cli/util/upload_bin.go

```diff
@@ -281,20 +281,16 @@ func searchIndexFile(ctx *cli.Context, p poolWrapper, containerID cid.ID, privKe
 		filters.AddFilter("IndexSize", fmt.Sprintf("%d", indexFileSize), object.MatchStringEqual)
 		for i := 0; ; i++ {
 			indexIDs := searchObjects(ctx.Context, p, containerID, privKeys, attributeKey, uint(i), uint(i+1), 1, maxRetries, debug, errCh, filters)
-			count := 0
-			for range indexIDs {
-				count++
-				if count > 1 {
-					select {
-					case errCh <- fmt.Errorf("duplicated index file %d found", i):
-					default:
-					}
-					return
-				}
+			resOIDs := make([]oid.ID, 0, 1)
+			for id := range indexIDs {
+				resOIDs = append(resOIDs, id)
 			}
-			if count == 0 {
+			if len(resOIDs) == 0 {
 				break
 			}
+			if len(resOIDs) > 1 {
+				fmt.Fprintf(ctx.App.Writer, "WARN: For index %d found %d index files with OIDs: %v\n", i, len(resOIDs), resOIDs)
```

Comment:
`%v` will give you an unreadable garbage. Use string representation, convert them manually.

Reply:
it won't.
```
(base) ekaterinapavlova@MacBook-Air-4 neo-go % ./bin/neo-go util upload-bin --cid 3RCdP3ZubyKyo8qFeo7EJPryidTZaGCMdUjqFJaaEKBV --wallet-config ./wallet-config.yml --block-attribute Block --index-attribute Index --rpc-endpoint https://rpc10.n3.nspcc.ru:10331 -fsr st4.storage.fs.neo.org:8080 -fsr st2.storage.fs.neo.org:8080 -fsr st3.storage.fs.neo.org:8080 -fsr st1.storage.fs.neo.org:8080 --retries 15 --searchers 1000 --workers 500
Chain block height: 6517131
WARN: For index 22 found 2 index files with OIDs: [73AsZJzibuRoyAvTgH1KN3NgrzbUAvuCiyEqjnYVLkYm 9zfEqjhqsm9xaKtCL6uYVB21mrVBNnYY7rnrAFWPYdAg]
^C
```